jupyter-extension/src/plugins/mount/__tests__/mount.test.tsx

```diff
@@ -96,15 +96,14 @@ describe('mount plugin', () => {
         name: 'default_images',
         project: 'default',
         branch: 'master',
-        commit: null,
+        commit: 'null',
```

Comment:
`'null'` seems strange, is this intended?

Reply:
Good catch! That was unintentional